docs/src/dedicated-gen-2/overview.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Platform.sh Dedicated Generation 2"
```

Comment:
Final naming needs to be decided upon.  Original idea was that existing **dedicated** becomes **legacy dedicated** and **dedicated** becomes the name for **dedication**. Marketing's suggestion is to keep current **dedicated** as is, and use **dedicated generation 2** for dedication…